cloudify_cli/commands/agents.py

```diff
@@ -44,23 +57,6 @@ def agents():
     pass
 
 
-def _is_deployment_installed(client, deployment_id):
```

Comment:
This used to test that all node instances are in `started` state.
In fact, only `Compute` instances need to have this check done.
The corrected test has been moved to the manager side (see https://github.com/cloudify-cosmo/cloudify-common/pull/216)